modApiServer/src/org/aion/api/server/http/nano/NanoRpcServer.java

```diff
@@ -93,24 +90,42 @@ public void makeSecure() throws Exception {
     @Override
     public void start() {
         try {
-            // default to 1 thread to minimize resource consumption by nano http
-            int tCount = 1;
+            /**
```

Comment:
should be comment `/*` not java doc `/**`

Reply:
fixed